syntax = "proto3";
package plugin;




import "agent_admin_sdk/model/agent_admin/agent.proto";

/*
ListPluginAgents请求
*/
message ListPluginAgentsRequest {
    /*
    页码
    */
    int32 page = 1 ;
    /*
    每页数据量
    */
    int32 pageSize = 2 ;
    /*
    ID
    */
    string id = 3 ;
}

/*
ListPluginAgents返回
*/
message ListPluginAgentsResponse {
    /*
    页数
    */
    int32 page = 1 ;
    /*
    页大小
    */
    int32 page_size = 2 ;
    /*
    总数
    */
    int32 total = 3 ;
    /*
    数据列表
    */
    repeated agent_admin.Agent list = 4 ;
}

/*
ListPluginAgentsApi返回
*/
message ListPluginAgentsResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    plugin.ListPluginAgentsResponse data = 4 ;
}

